<?xml version="1.0" encoding="UTF-8"?>
<launch>
 
  <!-- Load definition of pregrasp motion into play_motion namespace -->
  <rosparam command="load" file="$(find sim_tiago_kitchen)/config/pick_motions.yaml" /> 

  <node name="tiago_pick_and_place" pkg="sim_tiago_kitchen" type="tiago_pick_and_place" output="screen"/>
</launch>


